os/mm/mm_heap/mm_check_heap_corruption.c

```diff
@@ -84,14 +86,32 @@ typedef enum node_type_e node_type_t;
 
 static void dump_node(struct mm_allocnode_s *node, node_type_t type)
 {
+#if CONFIG_TASK_NAME_SIZE > 0
+#if defined(CONFIG_BUILD_PROTECTED) && !defined(__KERNEL__)
+	char myname[CONFIG_TASK_NAME_SIZE];
```

Comment:
Please declare the size with `CONFIG_TASK_NAME_SIZE + 1`.
You can refer to `tcb_s`.

Reply:
Done.